### DISCUSSION TITLE
wired ethernet

### DISCUSSION THREAD

**Discussion by williamjeccles** [macxq/foxess-ha#52] [Ideas]
![image](https://user-images.githubusercontent.com/6324545/157960491-0e306ff3-0e11-4857-84e7-006753c60163.png)

Could we somehow capture the data going over a hard-wired connection?

**Comment by ajmayers** [comment]
This is covered here in the HA-FoxESS-Modbus project and is close to being implemented:
https://github.com/StealthChesnut/HA-FoxESS-Modbus/discussions/25
